src/main/java/com/epam/esm/entity/Tag.java

```diff
@@ -1,12 +1,18 @@
 package com.epam.esm.entity;
 
-import org.springframework.stereotype.Component;
+import org.springframework.hateoas.RepresentationModel;
 import java.util.Objects;
+import javax.persistence.*;
 
-@Component
-public class Tag implements Identifiable {
+@Entity(name = "Tag")
+@Table(name = "tag")
+public class Tag extends RepresentationModel<Tag> implements Identifiable {
```

Comment:
Why `RepresentationModel<Tag>`?

Reply:
@a1exanddrovich how do you think is it okay to extend entity class in this way? Taking into the consideration that all its data is going to persisted

Reply:
do I have to create kinda a dto for all the entities?